src/SAML2/XML/init/RequestInitiator.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SAML2\XML\init;
+
+use DOMElement;
+use SAML2\XML\md\AbstractEndpointType;
+use Webmozart\Assert\Assert;
+
+/**
+ * Class for handling the init:RequestInitiator element.
+ *
+ * @link: http://docs.oasis-open.org/security/saml/Post2.0/sstc-request-initiation-cs-01.pdf
+ *
+ * @author Tim van Dijen, <tvdijen@gmail.com>
+ * @package simplesamlphp/saml2
+ */
+final class RequestInitiator extends AbstractEndpointType
+{
+    /** @var string */
+    public const NS = 'urn:oasis:names:tc:SAML:profiles:SSO:request-init';
+
+    /** @var string */
+    public const NS_PREFIX = 'init';
+
+
+    /**
+     * Set the value of the Binding property.
+     *
+     * @param string $binding
+     * @throws \InvalidArgumentException if the Binding is empty
+     */
+    protected function setBinding(string $binding): void
```

Comment:
Considering the binding is fixed, does it make sense having a `setBinding()` method?

Reply:
Well the alternative is overriding the constructor.. I thought it would make sense to keep the constructor the same as for other AbstractEndpoints